DNN Platform/Library/Security/Membership/AspNetMembershipProvider.cs

```diff
@@ -1874,6 +1874,13 @@ public override void UpdateUsersOnline(Hashtable userList)
                     //Clear cache for user so that locked out & other status could be updated
                     DataCache.ClearUserCache(portalId, username);
                     DataCache.ClearCache(GetCacheKey(username));
+
+                    aspnetUser = System.Web.Security.Membership.GetUser(username);
+
+                    if (aspnetUser.IsLockedOut)
+                    {
+                        loginStatus = UserLoginStatus.LOGIN_USERLOCKEDOUT;
+                    }
```

Comment:
@Mhtshum Please add a comment in code explaining the intent of this block.
It seems that after the 5th attempt the membership API is returning "invalid login" instead of "account locked", but then after the sixth it returns "account locked". So if that is the case, then I'm fine with this change but please add comment.

Reply:
@daguiler No, It would show Locked out message as soon as account is locked to default invalid attempts (i.e on 5th).
I'll add comment for this